src/GithubScrapper.php

```diff
@@ -36,4 +40,27 @@ public function getRepositoryURL(): string
     {
         return "https://github.com/" . $this->name;
     }
+
+    public function getWebsite(): string
+    {
+        $node_web = $this->crawler->filterXPath("//a[@rel='nofollow']")->first();
+        return $node_web->count() ? $node_web->html() : "unknown";
+    }
+
+    public function getTwitter(): string
+    {
+        return "https://twitter.com/" . $this->name;
```

Comment:
This won't be true if Twitter handle would be different than Github organization name.